assets/js/view/spreadsheet.rb

```diff
@@ -31,7 +31,24 @@ def render_table
       # TODO: consider adding train availability
     end
 
+    def render_or_history_titles(arbitrary_corporation)
```

Comment:
render_history(corporation)

Reply:
but that is not what this is. this generates the OR1.1, OR2.1, OR2.2 etc information for the title

it could be render_history_titles though, or something else

Reply:
sounds good